Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse.csproj

```diff
@@ -15,9 +15,9 @@
     <PackageTags>Frends</PackageTags>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <Description>Task for parsing fixed flat files.</Description>
+    <Description>Frends Task for parsing fixed flat files.</Description>
     <PackageProjectUrl>https://frends.com/</PackageProjectUrl>
-	<RepositoryUrl>https://github.com/FrendsPlatform/Frends.FixedWidthFlatFile/tree/main/Frends.FixedWidthFlatFile.Parse</RepositoryUrl>
+	<RepositoryUrl>https://github.com/FrendsPlatform/Frends.FixedWidthFlatFile</RepositoryUrl>
```

Comment:
Why was this changed? Don't we want the correct Task Repository Url to the csproj file?

Reply:
Reverted. It was made to match the another Task in this repo.